src/sagemaker_xgboost_container/encoder.py

```diff
@@ -18,12 +18,18 @@
 import tempfile
 from typing import Iterable
 
+import mlio
 import numpy as np
 import xgboost as xgb
 
 from sagemaker_containers import _content_types, _errors
 from sagemaker_xgboost_container.constants import xgb_content_types
 
+from mlio.integ.numpy import as_numpy
+from mlio.integ.scipy import to_coo_matrix
```

Comment:
Why the discrepancy in the naming of the two methods i.e. why not `as_coo_matrix` or `to_numpy`?

Reply:
Can @cbalioglu provide some insight?